src/commands/test-branch.yml

```diff
@@ -36,9 +36,9 @@ steps:
   - restore_cache:
       name: 'Solidus <<parameters.branch>>: Restore Bundler cache'
       keys:
-        - gems-v3-ruby-{{ checksum "<<parameters.working_directory>>/.ruby-version" }}-solidus-<<parameters.branch>>-{{ .Branch }}-{{ checksum "<<parameters.working_directory>>/Gemfile.lock" }}
-        - gems-v3-ruby-{{ checksum "<<parameters.working_directory>>/.ruby-version" }}-solidus-<<parameters.branch>>-master
-        - gems-v3-ruby-{{ checksum "<<parameters.working_directory>>/.ruby-version" }}-solidus-<<parameters.branch>>-
+        - gems-v3.{{ .Environment.CACHE_VERSION }}-ruby-{{ checksum "<<parameters.working_directory>>/.ruby-version" }}-solidus-<<parameters.branch>>-{{ .Branch }}-{{ checksum "<<parameters.working_directory>>/Gemfile.lock" }}
```

Comment:
total nit:

```
gems-v3.
       ^___ this dot
```

should we use dots as separator for the whole key?

Reply:
My intention was only to use it for the version part so that it could be read as `v#{code_part}.{env_part}`. WDYT?

Reply:
I'll merge to move forward, but happy to keep talking about it and submit another PR if needed.

Reply:
good call 👍 wasn't important